src/PIL/Image.py

```diff
@@ -1104,12 +1104,9 @@ def _crop(self, im, box):
 
         x0, y0, x1, y1 = map(int, map(round, box))
 
-        if x1 < x0:
-            x1 = x0
-        if y1 < y0:
-            y1 = y0
+        absolute_values = (abs(x1 - x0), abs(y1 - y0))
```

Comment:
left the parentheses since _decompression_bomb_check expects a tuple (Image.size), added spaces around minuses

Reply:
It's a tuple with or without parentheses:

```
⌂86% [hugo:~/github/pypistats] master* ± p
Python 3.7.0 (default, Jun 29 2018, 20:13:13)
[Clang 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> absolute_values = (1, 2)
>>> type(absolute_values)
<class 'tuple'>
>>> absolute_values = 1, 2
>>> type(absolute_values)
<class 'tuple'>
>>>
⌂140% [hugo:~/github/pypistats] master* 21s ± p2
Python 2.7.15 (default, Jun 17 2018, 12:46:58)
[GCC 4.2.1 Compatible Apple LLVM 9.1.0 (clang-902.0.39.2)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> absolute_values = (1, 2)
>>> type(absolute_values)
<type 'tuple'>
>>> absolute_values = 1, 2
>>> type(absolute_values)
<type 'tuple'>
>>>
```

Reply:
Thanks for the suggestion, didn't know that.